content/english/android-tictactoe/_index.md

```diff
@@ -7,14 +7,14 @@ difficulty: "Intermediate"
 download: "resources/_gen/files/TicTacToe.zip"
 icon: "fab fa-android"
 draft: false
-hidden: true
+hidden: false
```

Comment:
@ozhang22 Do you want alll of these set back to hidden for this PR?

Reply:
@sw23 Actually, I think it's fine to make this all unhidden.